Sources/PackageLoading/PackageDescription4Loader.swift

```diff
@@ -17,7 +17,8 @@ extension ManifestBuilder {
         let package = try json.getJSON("package")
         self.name = try package.get("name")
         self.pkgConfig = package.get("pkgConfig")
-        let slv = try? package.get([String].self, forKey: "swiftLanguageVersions")
+        let slv = (try? package.get([String].self, forKey: "swiftLanguageVersions")) ??
+            (try? package.get([Int].self, forKey: "swiftLanguageVersions").map(String.init))
```

Comment:
Why Int?

Reply:
Because under PD < 4.2, `swiftLanguageVersions` is of type `[Int]?`. Customizing the decoding function to return a `[String]` in that case would require more code than to support decoding `[Int]` here. Thoughts?

Reply:
Do you mean that we'll have to implement `encode(to:)` method for the `Package` object? We should do that anyway..